src/main/webapp/vendor/startbootstrap/template.jsp

```diff
@@ -131,7 +132,7 @@
         <footer class="sticky-footer bg-white">
           <div class="container my-auto">
             <div class="copyright text-center my-auto">
-              <span>Copyright &copy; Capstone-Filler</span>
+              <span>Copyright &copy; scheduli</span>
```

Comment:
this actually needs to be "Google LLC", since scheduli isn't a registered entity of any sort.